docs/source/using/troubleshooting.adoc

```diff
@@ -314,3 +314,28 @@ You can configure the URL by running:
 $ minishift config set check-network-http-host <URL>
 ----
 
+[[github-api-rate-limit-exceeded]]
+== Github API rate limit exceeded
+
+When you run `minishift start` or `minishift update`, it could make a request to Github to get the latest tarball of Minishift or Openshift client.
+Sometimes, during these requests, you may receive a 403 forbidden status from Github if your network has reached API rate limit and the command does not succeed, like the one below:
+
+----
+Error starting the cluster: Error attempting to download and cache oc: Cannot get the OpenShift
+release version v3.6.0: GET https://api.github.com/repos/openshift/origin/releases/tags/v3.6.0:
+403 API rate limit exceeded for (your IP shows here). (But here's the good news: Authenticated
+requests get a higher rate limit. Check out the documentation for more details.); rate reset in
+17m2.462768522s
+----
+
+Github has a rate limiting policy, see link:https://developer.github.com/v3/#rate-limiting[Rate Limiting]. You may have reached this limit for many reasons - your package manager may use Github API calls or you are behind a corporate network that makes a lot of Github unauthenticated API calls.
+
+Instead of waiting Github to reset the limit for your network so you can run the commands successfully, you can create a personal API token from your Github account as the rate limit for authenticated requests are higher, see link:https://github.com/blog/1509-personal-api-tokens[Personal API Tokens] for more details on how to generate an API token.
```

Comment:
* waiting > waiting for
* as the rate > that determines the rate
* replace comma with period before "see Personal API Tokens"